html/editing/focus/focus-02.html

```diff
@@ -7,12 +7,8 @@
 <meta assert="assert" content="Check if the key events received by document are targeted at the element when no element is focused">
 <script src="/resources/testharness.js"></script>
 <script src="/resources/testharnessreport.js"></script>
-<h2>Steps:</h2>
```

Comment:
Do we have some convention about leaving manual steps in here so that they can be run without testdriver?

Reply:
Not really. I think leaving the alone where they exist would be OK, but I wouldn't expect the instructions to stay working over time, or that they'd be added for new tests, so I'd perhaps slightly prefer removing them. WDYT?

Reply:
I think they should be removed.
When new tests are added with testdriver they generally aren't accompanied by manual steps incase they are run without testdriver, so I think we should stick to this convention. I also agree with what @foolip says with maintenance, but @jgraham you bring up a good point that makes me wonder whether we should even allow testdriver tests to be run in any other way than wpt run. I opened #10330 to further address this.